packages/text-field/index.tsx

```diff
@@ -45,6 +45,8 @@ export interface Props<T> {
   lineRippleClassName: string;
   notchedOutlineClassName: string;
   outlined: boolean;
+  onLeadingIconSelect?: () => void;
```

Comment:
See my previous response about why I did not include a default prop for onSelect. This would hoist the problem. Now renderIcon will always render with an onSelect and would be problematic for setting the tabindex.

Reply:
sounds good!